src/app/app.component.spec.ts

```diff
@@ -20,13 +39,13 @@ describe('AppComponent', () => {
     it(`should have as title 'app'`, async(() => {
         const fixture = TestBed.createComponent(AppComponent);
         const app = fixture.debugElement.componentInstance;
-        expect(app.title).toEqual('app');
+        expect(app.title).toEqual('PC Prep Kit');
     }));
 
-    it('should render title in a h1 tag', async(() => {
-        const fixture = TestBed.createComponent(AppComponent);
-        fixture.detectChanges();
-        const compiled = fixture.debugElement.nativeElement;
-        expect(compiled.querySelector('h1').textContent).toContain('Welcome to app!!');
-    }));
+    // it('should render title in a h1 tag', async(() => {
+    //     const fixture = TestBed.createComponent(AppComponent);
+    //     fixture.detectChanges();
+    //     const compiled = fixture.debugElement.nativeElement;
+    //     expect(compiled.querySelector('h1').textContent).toContain('Welcome to app!!');
+    // }));
```

Comment:
Can this be removed if its not used?

Reply:
Okay! I'll remove all the commented code. :+1: